liam2/simulation.py

```diff
@@ -241,6 +242,12 @@ def from_str(cls, yaml_str, simulation_dir='',
         #                'MIG': {'type': int}}
         globals_def = {}
         for k, v in content.get('globals', {}).iteritems():
+            if 'value' in v:
+                if np.isscalar(v['value']) or isinstance(v['value'], bool):

```

Comment:
* unsure restricting to scalars adds any value
* isinstance(v['value'], bool) seems useless